.vscode/settings.json

```diff
@@ -11,6 +11,8 @@
         "--manifest-path",
         "kernel/Cargo.toml",
         "--target",
+        "riscv64gc-unknown-none-elf",
+        "--target",
         "x86_64-unknown-none",
```

Comment:
Does this change mean `cargo check` will check if any compilation errors for both targets? 

Could you move x86-64 before RISC-V? We want to order the supported CPU architectures according to the importance. Currently, x86-64 is Asterinas and OSDK's Tier 1 target, while RISC-V is Tier 2.

Reply:
> Does this change mean `cargo check` will check if any compilation errors for both targets?

Yes, the rust analyzer checks two targets. However, there is a drawback that x86 developers will also see RISC-V compile warnings, making development on x86 a little more difficult.

I think we can make this PR a lower priority. Let's refine the RISC-V implementation and address all warnings before merging this PR.